src/api/app/mixins/forbids_anonymous_users.rb

```diff
@@ -8,9 +8,9 @@ class AnonymousUser < APIException
   # conditionally loading the anonymous user!
   #
   def require_login
-    # we may allow anonymous GET operations (if configured) but we require
-    # a valid account on other opertations
-    be_not_nobody! unless request.get?
+    # we allow anonymous user only for rare special operations (if configured) but we require
+    # a valid account for all other operations
+    be_not_nobody! unless /(\/public|\/trigger|\/about)/ =~ request.env['PATH_INFO']
```

Comment:
Might not be a problem in most cases, but ^ would be the beginning of a newline. \A and \Z would match start and end of a string:)